src/lib/Scenes/Artwork/Components/ImageCarousel/ImageCarousel.tsx

```diff
@@ -0,0 +1,111 @@
+import { color, Flex, space, Spacer } from "@artsy/palette"
+import { ImageCarousel_images } from "__generated__/ImageCarousel_images.graphql"
+import { devCacheBust } from "lib/utils/devCacheBust"
+import React, { useCallback, useEffect, useMemo, useState } from "react"
+import { Animated, Dimensions, FlatList, NativeScrollEvent, NativeSyntheticEvent } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+import { findClosestIndex, getMeasurements } from "./geometry"
+import { ImageWithLoadingState } from "./ImageWithLoadingState"
+
+export interface ImageCarouselProps {
+  images: ImageCarousel_images
+}
+
+const windowWidth = Dimensions.get("window").width
+// The logic for cardHeight comes from the zeplin spec https://zpl.io/25JLX0Q
+const cardHeight = windowWidth >= 375 ? 340 : 290
+export const cardBoundingBox = { width: windowWidth, height: cardHeight }
+
+export const ImageCarousel: React.FC<ImageCarouselProps> = ({ images }) => {
```

Comment:
Per the convo in slack, mind adding a comment here with a short description of what a future developer should keep in mind if they need to extract this for other use-cases?